infobip-typescript-generator-extension-annotation-processor/src/main/java/com/infobip/typescript/TypescriptAnnotationProcessor.java

```diff
@@ -52,9 +65,11 @@ private void generateTypeScript(TypeElement element) {
     private TypeScriptFileGenerator createTypeScriptFileGenerator(TypeElement element,
                                                                   GenerateTypescript generateTypescript) {
         Path basePath = getBasePath(element);
+        Path decoratorsBasePath = basePath.getParent().getParent().resolve("src/main/typescript/decorators");
```

Comment:
This is added to position code in direcotry on disk but not in target.
Reason for that is explain in comment for pom.xml, mvn clean install firs cleans everything in targets and after that runs AnnotationProcessor.
Code will fail because there are no resources in target, because of clean command.
With this approach we can access decorators directly on disk, and makes configuration for resource folder in pom.xml not needed anymore.

Reply:
old comment logic is now changed